development/templates/development/match_details.html

```diff
@@ -4,6 +4,14 @@
 <div class="container-fluid">
     {% if data %}
     <h1>Match details 
+
+        <select class="form-control" name="valid_move" id="valid_move">
```

Comment:
All user inputs MUST have a label `<label for="id_of_input">Label Text</label>` to give context to accessibility tools

Reply:
Excellent I didn't know that. Than you